package.json

```
@@ -12,7 +12,7 @@
     "packages/*"
   ],
   "devDependencies": {
-    "@appsignal/types": "^2.0.2",
+    "@appsignal/types": "^2.1.1",
```

```suggestion
    "@appsignal/types": "^2.1.2",
```

2.1.2 is the last version right? Can you update and also commit the updated lock file?